pkgs/games/itch/default.nix

```diff
@@ -1,30 +1,53 @@
 { lib
 , stdenvNoCC
-, fetchurl
+, fetchzip
 , libnotify
 , nss
 , gtk3
+, electron_11
+, steam-run
 , fetchFromGitHub
+, makeWrapper
+, copyDesktopItems
 , makeDesktopItem
-, itch-setup
-, runtimeShell
 }:
 stdenvNoCC.mkDerivation rec{
   pname = "itch";
   version = "25.5.1";
 
-  src = fetchFromGitHub {
+  src = fetchzip {
+    stripRoot = false;
+    url = "https://broth.itch.ovh/${pname}/linux-amd64/${version}/itch.zip";
+    sha256 = "sha256-ejfS+sqhacW2h8u96W4fout3V8xrBs0SrW5w/7X83m4=";
+  };
+  patches = [ ./itch.patch ];
+  butler = fetchzip {
+    stripRoot = false;
+    url = "https://broth.itch.ovh/butler/linux-amd64/15.21.0/butler.zip";
+    sha256 = "sha256-jHni/5qf7xST6RRonP2EW8fJ6647jobzrnHe8VMx4IA=";
+  };
+  itch-setup = fetchzip {
```

Comment:
We already have packaged itch-setup in nixpkgs

Reply:
You are right.
I'm doing this because in opinion `itch-setup` isn't something that makes much sense to have in Nixpkgs and I was planning to remove it. (And I've done it with the last push.)
Of course, if you are opposed to this decision, tell me and I'll revert it.

Reply:
I think that, for example, something like Butler makes more sense to be packaged.

Reply:
If the build still uses itch-setup then it is better to leave it in my opinion. Also the supported way to run itch is via itch setup so it may be a good idea to keep a `itch-bin` that run as it is now.

Reply:
So, you suggest me to leave `itch` as it is, and make this PR for a separate `itch-bin`?

Reply:
You don't "build from source" an Electron app, what the Itch developers do to create the "binary package" (in quotes because it's not binary) are things like converting TypeScript to JavaScript, there is no point doing this sort of stuff in Nixpkgs.
Every Electron package on Nixpkgs (and I suppose the AUR too) fetches the "binary" directly.

Reply:
https://github.com/NixOS/nixpkgs/blob/nixos-21.11/pkgs/applications/networking/browsers/vieb/default.nix is a electron app compiled from source.

Also it may be semantics but I do consider bundling a compilation step.

Reply:
The package name should be itch, please use [source provenance](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#source-provenance-sec-meta-sourceprovenance) to indicate that it is a binary.
Building electron things from source is something we generally avoid.

Reply:
I added the source provenance, though searching which packages define it yields no result.
I didn't know it existed...